swat/cas/table.py

```diff
@@ -7798,20 +7796,20 @@ def capitalize(self):
 
     def contains(self, pat, case=True, flags=0, na=np.nan, regex=True):
         '''
-        Does the value contain the specified pattern?
+        Indicates wheter the value contains the specified pattern.
```

Comment:
No period at end.  This should be done for all of the one-line descriptions of methods.

Reply:
Fixed for all.